frigate/api/new_app.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from fastapi import FastAPI
+
+from frigate.api import app as main_app
+from frigate.api import media, preview
+
+logger = logging.getLogger(__name__)
+
+# https://fastapi.tiangolo.com/tutorial/metadata/#use-your-tags
+tags_metadata = [
+    {
+        "name": "Preview",
+        "description": "Preview routes",
+    },
+    {
+        "name": "Logs",
+        "description": "Logs routes",
+    },
+    {
+        "name": "Media",
+        "description": "Media routes",
+    },
+]
+
+
+def create_fastapi_app(frigate_config, detected_frames_processor):
```

Comment:
this is good to see, I was not sure how hard it would be to map these additional objects into the api class so they would be available to the handlers